src/Integration.UnitTests/LocalServices/UnboundProjectFinderTests_ProjectLevelConfig.cs

```diff
@@ -1,348 +0,0 @@
-using System.Linq;
-using EnvDTE;
-using FluentAssertions;
-using Microsoft.VisualStudio.CodeAnalysis.RuleSets;
-using Microsoft.VisualStudio.TestTools.UnitTesting;
-using Moq;
-using SonarLint.VisualStudio.Core.Binding;
-using SonarLint.VisualStudio.Integration.Binding;
-using SonarLint.VisualStudio.Integration.NewConnectedMode;
-using Language = SonarLint.VisualStudio.Core.Language;
-
-// This file contains tests for UnboundProjectFinder that relate to
-// the project-level config files.
-// The solution-level config tests are in another class.
-
-namespace SonarLint.VisualStudio.Integration.UnitTests
-{
-    [TestClass]
-    public class UnboundProjectFinderTests_ProjectLevelConfig
-    {
-        private ConfigurableServiceProvider serviceProvider;
-        private ConfigurableConfigurationProvider configProvider;
-        private ConfigurableRuleSetSerializer ruleSetSerializer;
-        private ConfigurableVsProjectSystemHelper projectSystemHelper;
-        private ConfigurableSolutionRuleSetsInformationProvider ruleSetInfoProvider;
-        private Mock<IFileSystem> fileMock;
-
-        [TestInitialize]
-        public void TestInit()
-        {
-            this.serviceProvider = new ConfigurableServiceProvider();
-
-            this.configProvider = new ConfigurableConfigurationProvider();
-            this.serviceProvider.RegisterService(typeof(IConfigurationProvider), this.configProvider);
-
-            this.projectSystemHelper = new ConfigurableVsProjectSystemHelper(this.serviceProvider);
-            this.serviceProvider.RegisterService(typeof(IProjectSystemHelper), this.projectSystemHelper);
-
-            this.ruleSetInfoProvider = new ConfigurableSolutionRuleSetsInformationProvider();
-            this.serviceProvider.RegisterService(typeof(ISolutionRuleSetsInformationProvider), this.ruleSetInfoProvider);
-
-            this.ruleSetSerializer = new ConfigurableRuleSetSerializer(new MockFileSystem());
-            this.serviceProvider.RegisterService(typeof(IRuleSetSerializer), this.ruleSetSerializer);
-
-            this.fileMock = new Mock<IFileSystem>();
-            fileMock.Setup(x => x.File.Exists(It.IsAny<string>())).Returns(false);
-        }
-
-        #region Tests
-
-        [TestMethod]
-        public void ArgCheck()
-        {
-            Action action = () => new UnboundProjectFinder(null, new ProjectBinderFactory(serviceProvider, Mock.Of<ILogger>()));
-            action.Should().ThrowExactly<ArgumentNullException>().And.ParamName.Should().Be("serviceProvider");
-
-            action = () => new UnboundProjectFinder(serviceProvider, projectBinderFactory: null);
-            action.Should().ThrowExactly<ArgumentNullException>().And.ParamName.Should().Be("projectBinderFactory");
-        }
-
-        [TestMethod]
-        [DataRow(SonarLintMode.Connected)]
-        [DataRow(SonarLintMode.LegacyConnected)]
-        public void GetUnboundProjects_ValidSolution_ProjectRuleSetIsMissing(SonarLintMode mode)
```

Comment:
Deleted, the tests now use mocks and all the relevant testing is part of CSharpVBProjectBinderTests